docs/macios/xcsync.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Xcode Sync
+description: Xcsync is designed to provide .NET developers with improved support for editing Apple specific files
+author: haritha-mohan; mcumming
+ms.author: harithamohan; mcumming
+ms.date: 08/05/2024
+monikerRange: ">=net-maui-9.0"
+---
+# Xcode Sync (Xcsync)
+
+Xcode Sync (Xcsync) is a tool that enables developers to leverage Xcode for managing Apple specific files with .NET projects. The tool generates a temporary Xcode project from a .NET project and synchronizes changes to the Xcode files back to the .NET project.
+
+Supported file types include:
+
+- Asset Catalog
+- Plist
+- Storyboard
+- Xib
+
+The tool has 2 commands: `generate` and `sync`. Use `generate` to create an Xcode project from a .NET project and `sync` to bring changes in the Xcode project back to the .NET project.
+
+## Synopsis
+
+### xcsync-generate
+
+```dotnetcli
+dotnet build /t:xcsync-generate
+    /p:xcSyncProjectFile=<PROJECT>
+    /p:xcSyncXcodeFolder=<TARGET_XCODE_DIRECTORY>
+    /p:xcSyncTargetFrameworkMoniker=<FRAMEWORK>
+    /p:xcSyncVerbosity=<LEVEL>
```

Comment:
This isn't about the docs, but the tool itself: `dotnet build` needs a project to actually build, which I believe is the same as `xcSyncProjectFile`, so doesn't that make `xcSyncProjectFile` redundant?

So:

```suggestion
dotnet build /t:xcsync-generate <PROJECT>
    /p:xcSyncXcodeFolder=<TARGET_XCODE_DIRECTORY>
    /p:xcSyncTargetFrameworkMoniker=<FRAMEWORK>
    /p:xcSyncVerbosity=<LEVEL>
```

And then we can just default `xcSyncProjectFile=$(MSBuildProjectFullPath)` in the `xcsync-generate`/`xcsync-sync` targets.

Reply:
yea we looked into this but dotnet build technically accepts either a project or solution parameter, but if a solution path is passed to xcsync it will break..so to avoid any potential confusion here at least for the initial preview we decided to explicitly specify. but this is definitely something we can refactor prior to the next release

Reply:
One point is that if you don't pass a project to `dotnet build`, it must exist in the current directory, and there can't be any other project files in that directory. This isn't entirely obvious from the help here (I could certainly see myself trying to run `dotnet build ...` from my HOME directory by accident, and that won't work - I find the problem and solution to be somewhat obvious, but that doesn't mean other people won't scratch their heads a bit).

Reply:
yea fair enough, that's a good point it would be good to have better clarity here. I'll keep that in mind when updating docs for RC.